clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleLoadBalancerCachingAgent.groovy

```diff
@@ -193,12 +193,16 @@ class GoogleLoadBalancerCachingAgent extends AbstractGoogleCachingAgent implemen
         loadBalancers << newLoadBalancer
 
         if (forwardingRule.target) {
-          def targetPoolName = Utils.getLocalName(forwardingRule.target)
-          def targetPoolsCallback = new TargetPoolCallback(googleLoadBalancer: newLoadBalancer,
-                                                           httpHealthChecksRequest: httpHealthChecksRequest,
-                                                           instanceHealthRequest: instanceHealthRequest)
+          def forwardingRuleTokens = forwardingRule.target.split("/")
 
-          compute.targetPools().get(project, region, targetPoolName).queue(targetPoolsRequest, targetPoolsCallback)
+          if (forwardingRuleTokens[forwardingRuleTokens.size() - 2] != "targetVpnGateways") {
```

Comment:
Worth pointing out here that I'd rather we had a more formalized way to determine the type of the thing being targeted, but I'm unable to find one. So we're stuck with string-work.